webpack.config.js

```diff
@@ -1,12 +1,12 @@
-var webpack = require('webpack');
-var path = require('path');
-var fs = require('fs');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const path = require('path');
+const fs = require('fs');
 
-var config = [{
+const config = [{
   entry: {
     './htdocs/js/components/DynamicDataTable.js': './jsx/DynamicDataTable.js',
-    './htdocs/js/components/PaginationLinks.js': './jsx/PaginationLinks.js',
-    './htdocs/js/components/StaticDataTable.js': './jsx/StaticDataTable.js',
+    './htdocs/js/components/PaginationLinks.js': ['./jsx/PaginationLinks.js'],
```

Comment:
why do these have square brackets but not the others ??

Reply:
@ridz1208 The square brackets solves an error for the frontend when importing the StaticDataTable component. 

The error message "Module not found: Error: a dependency to an entry point is not allowed" will be thrown when running "npm run compile" without the brackets when importing the StaticDataTable into a component.

Supposedly the latest version of React solves the issue but we're still using an older version that requires this fix. Also to note the square brackets are usually used to compile an array of jsx files into one javascript file to serve to the client browser instead of multiple js files.